server/plugin_release.go

```diff
@@ -128,6 +139,7 @@ func getReleaseByTag(ctx context.Context, client *GithubClient, owner, repositor
 }
 
 // createTag creates a new tag at the given commit for the repository.
+// Also marks the release with the given tag as pre-release if the preRelease flag is set true
```

Comment:
Does it? It looks like you have to do that separately.

Reply:
Hmm, this is a stale comment that I apparently forgot to strip out from the previous iteration. Fixing as:
```suggestion
```